app/src/org/commcare/connect/ConnectIdWorkflows.java

```diff
@@ -573,7 +573,7 @@ private static ConnectTask completeUnlock() {
         return ConnectTask.CONNECT_NO_ACTIVITY;
     }
 
-    private static void completeSignIn() {
+    public static void completeSignIn() {
```

Comment:
This class is going away, so calling the callback will have to happen from somewhere else.